crates/re_viewer/src/ui/recordings_panel.rs

```diff
@@ -0,0 +1,104 @@
+use re_data_store::StoreDb;
+use re_log_types::Time;
+use re_viewer_context::{SystemCommand, SystemCommandSender, ViewerContext};
+use time::macros::format_description;
+
+#[cfg(not(target_arch = "wasm32"))]
+use re_ui::UICommandSender;
+
+/// Show the Recordings section of the left panel
```

Comment:
it's good to describe what the code does: "Shows the currently open recordings in a selectable list"